a:
  pose:
    x: 3.7
    y: 3.7
    yaw: 180
  connected_nodes:
    - "b"
    - "c"
    - "d"
    - "l"

b:
  pose:
    x: 3.1
    y: 3.623
    yaw: 90
  connected_nodes:
    - "a"
    - "c"
    - "d"
#    - "-e"
    - "e"
    - "l"
    - "m"

c:
  pose:
    x: 3.7
    y: 3.34
    yaw: -90
  connected_nodes:
    - "a"
    - "b"
    - "d"
    - "l"

d:
  pose:
    x: 3.7
    y: 3.0
    yaw: -90
  connected_nodes:
#    - "-a"
#    - "-c"
    - "a"
    - "c"

e:
  pose:
    x: 3.1
    y: 1.92
    yaw: -90
  connected_nodes:
#    - "-b"
#    - "-f"
    - "b"
    - "f"

f:
  pose:
    x: 3.1
    y: 0.748
    yaw: 0
  connected_nodes:
#    - "-e"
    - "e"
    - "g"

g:
  pose:
    x: 3.68
    y: 0.717
    yaw: 0
  connected_nodes:
#    - "-f"
    - "f"
    - "h"
    - "i"
    - "j"
    - "k"

h:
  pose:
    x: 3.65
    y: 0.56
    yaw: 180
  connected_nodes:
    - "i"
    - "g"
    - "j"
    - "k"

i:
  pose:
    x: 3.65
    y: 0.37
    yaw: 180
  connected_nodes:
    - "g"
    - "k"
    - "j"
    - "h"

j:
  pose:
    x: 3.7
    y: 1.0
    yaw: 90
  connected_nodes:
    - "g"
    - "h"
    - "i"
    - "k"

k:
  pose:
    x: 3.7
    y: 1.17
    yaw: 90
  connected_nodes:
    - "-j"
    - "-g"

l:
  pose:
    x: 2.21
    y: 3.5
    yaw: 90
  connected_nodes:
    - "a"
    - "b"
    - "m"
    - "n"
    - "o"

m:
  pose:
    x: 2.385
    y: 2.856
    yaw: -90
  connected_nodes:
    - "l"
    - "b"
    - "n"
    - "o"

n:
  pose:
    x: 2.385
    y: 1.248
    yaw: -90
  connected_nodes:
    - "l"
    - "m"
    - "o"
    - "p"
    - "q"

o:
  pose:
    x: 2.385
    y: 0.348
    yaw: -90
  connected_nodes:
    - "l"
    - "m"
    - "n"
    - "p"
    - "q"

p:
  pose:
    x: 1.549
    y: 0.352
    yaw: 90
  connected_nodes:
    - "n"
    - "o"
    - "q"
    - "r"

q:
  pose:
    x: 1.539
    y: 1.21
    yaw: 90
  connected_nodes:
    - "r"
    - "p"
    - "o"
    - "n"

r:
  pose:
    x: 1.492
    y: 1.598
    yaw: 90
  connected_nodes:
    - "t"
    - "q"
    - "p"
    - "s"

s:
  pose:
    x: 1.492
    y: 3.225
    yaw: 90
  connected_nodes:
    - "t"
    - "r"
    - "q"
    - "u"
    - "v"

t:
  pose:
    x: 1.47
    y: 3.5
    yaw: 90
  connected_nodes:
    - "s"
    - "u"
    - "v"

u:
  pose:
    x: 0.9
    y: 3.65
    yaw: 180
  connected_nodes:
    - "t"
    - "s"
    - "v"

v:
  pose:
    x: 0.35
    y: 2.826
    yaw: -90
  connected_nodes:
    - "u"
    - "t"
    - "s"
    - "w"

w:
  pose:
    x: 0.35
    y: 0.9
    yaw: -90
  connected_nodes:
    - "v"
    - "x"

x:
  pose:
    x: 0.889
    y: 0.59
    yaw: 90
  connected_nodes:
    - "w"